samples/sensor/hx711/README.rst

```diff
@@ -10,13 +10,37 @@ Offset is calculated without any weight put on the scale.
 Then the user is prompted to put the CONFIG_HX711_CALIBRATION_WEIGHT on the scale and
 calibration is performed.
 
-After that calibration the weight is printed every second.
+After calibration :
```

Comment:
all of the sample-related changes should be in commit `sensor: add example for HX711`. Some are incorrectly in commit `sensor: add driver for HX711`

Reply:
You are right. 
Fixed.